run.py

```diff
@@ -3,20 +3,26 @@
 """
 import sys
 import os
+import shutil
+
+path = "/home/user/NetPyNE-UI/workspace/NetPyNEShowcase/NeuroML2/tmp"
```

Comment:
Is it ok to have a full unix-like path hard-coded here? I have no idea if it's because it will be tightly linked to a docker image later, or if it should be changed for a path depending on the host machine :sweat_smile:

Reply:
@filippomc

Reply:
I need to fix this, thanks good catch @aranega!